packaging/docker-alfresco/Dockerfile

```diff
@@ -62,12 +64,7 @@ RUN sed -i -e "s_appender.rolling.fileName\=alfresco.log_appender.rolling.fileNa
 
 # fontconfig is required by Activiti worflow diagram generator
 # installing pinned dependencies as well
```

Comment:
We removed the installation of the dependencies, so we might remove this comment

Reply:
We didn't though: https://github.com/Alfresco/alfresco-community-repo/pull/2841/files#diff-3914a9b710eef3208cac88e83b2b8192575da4a48d058cfa21905c529883e0dcR67.

It's ok as is.